<launch>
  <param name="robot_description" command="$(find xacro)/xacro $(find cooking_robot)/urdf/primary_manipulator.xacro"/>
  <node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-param robot_description -urdf -model primary_manipulator"/>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="paused" value="true"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
  </include>
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" output="screen" />
  <rosparam file="$(find cooking_robot)/config/joint_trajectory_controller.yaml" command="load" />
  <node pkg="controller_manager" type="spawner" name="controller_spawner" args="primary_mnipulator_controller"/>
    

</launch>
